tools/generator/templates/champions/cxx/api.hh.jinja2

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2012-2020 Association Prologin <association@prologin.org>
+
+// This file contains all the API functions for the C++ language, and all the
+// constants, enumerations and structures.
+// It has been generated. You can modify the end of the file if you want, but
+// do not touch the part where constants, enums, structs, and api functions are
+// defined.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+{%- for constant in game.constant %}
```

Comment:
This could be factored in a cxx stub between the rules and the API.

Reply:
I used the cxx jinja macros